Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-register/ContestRegister.tsx

```diff
@@ -111,12 +118,41 @@ const ContestRegister = () => {
         registerUserForContest,
         navigate,
     ]);
+    const getErrorMessage = (err: FetchBaseQueryError | SerializedError, defaultErrorMessage: string): string => {
+        if ('status' in err) {
+            return 'error' in err
+                ? err.error.replace(/"/g, '')
+                : JSON.stringify((err.data as any).replace(/"/g, ''));
+        }
+        if (err.message) {
+            return err.message.replace(/"/g, '');
+        }
+
+        return defaultErrorMessage;
+    };
```

Comment:
Shouldn't this be common logic?

Reply:
what do you mean 'common' logic ?

Reply:
I mean extracted to be reused

Reply:
this is example of the logic i was talking to abstract for error handling throught the whole applicaiton. I made this implementation here since the BE retruns good error messages. I suggest the exraction of this logic to be made in another task where the whole error handling of the platform is made via such function and error component, but in this PR.